tests/test_build.py

```diff
@@ -23,3 +23,33 @@ def test_run_pymodules_install_optional_project_dir(self):
             assert run_pymodules_install(ctx, modules, project_dir) is None
         assert m_info.call_args_list[-1] == mock.call(
             'No Python modules and no setup.py to process, skipping')
+
+    def test_strip_if_debuggable(self):
+        ctx = mock.Mock()
```

Comment:
I'm surprised we're not using an actual `Context` object, but I guess why not :smile: